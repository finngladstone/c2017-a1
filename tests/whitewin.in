place A1
place A5
place A2
place B6
place C4
place C7
place A9
place D8
place C14
place E9
