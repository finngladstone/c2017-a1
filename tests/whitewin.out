White wins!
A1A5A2B6C4C7A9D8C14E9
Thank you for playing!
